src/layouts/product-integration-layout/index.tsx

```diff
@@ -126,18 +127,102 @@ export default function ProductIntegrationLayout({
 					<meta name="robots" content="noindex, nofollow" />
 				</HashiHead>
 			)}
-			<Header
-				productSlug={currentProduct.slug}
+
+			<ContentHeaderCard
 				className={s.header}
-				integration={integration}
-				activeRelease={activeRelease}
-				getVersionChangedURL={getVersionChangedURL}
-				onInstallClicked={() => {
-					console.log('TODO, probably remove this')
-				}}
+				icon={currentProduct.slug !== 'sentinel' ? currentProduct.slug : null}
+				title={integration.name}
+				attribution={`@${integration.organization.slug}`}
+				description={integration.description}
+				note={lastUpdatedString(integration.updated_at)}
+				badges={getIntegrationBadges(integration, true)}
+				dropdown={
+					integration.versions.length > 1
+						? {
+								text: versionString(
+									activeRelease.version,
+									integration.versions
+								),
+								items: integration.versions
+									.filter((e: string) => e !== activeRelease.version)
+									.map((version: string) => {
+										return {
+											text: versionString(version, integration.versions),
+											href: getVersionChangedURL(version),
+										}
+									}),
+						  }
+						: undefined
+				}
+				links={[
+					{
+						text: 'GitHub',
+						href: integration.subdirectory
+							? `${integration.repo_url}/tree/main${integration.subdirectory}`
+							: integration.repo_url,
+						icon: <IconGithub16 />,
+					},
+				]}
 			/>
+
 			<h1>{title}</h1>
```

Comment:
🔴 Just noticing this (probably out of scope for this PR) -- This ends up always rendering a second `h1` on the page, which we don't want to do. Can you create a ticket for resolving this? I'm not sure what board to place it on.

Reply:
I have a ticket for this one already:

[Remove potential Duplicative H1 from README](https://app.asana.com/0/1203792616809419/1203295231021660)

It has been bugging me for a little bit as well.  I've bumped up the priority on this one, thanks for the callout here.